### ISSUE TITLE
layer-wise learning rate decay (LLRD)

### ISSUE THREAD

**Issue by bwanglzu** [jina-ai/finetuner#690]
implement LLRD in finetuner, this has been a common technique to improve the fine-tuning efficiency on transformers, and has been proven to be useful for BERT and CLIP fine-tuning.

reference:
1.  Revisiting Few-sample BERT Fine-tuning: https://arxiv.org/abs/2006.05987
2. CLIP Itself is a Strong Fine-tuner: https://arxiv.org/pdf/2212.06138.pdf
